packages/ra-core/src/auth/useHandleLoginCallback.ts

```diff
@@ -0,0 +1,57 @@
+import { useQuery, UseQueryOptions } from 'react-query';
+import { useBasename, useRedirect } from '../routing';
+import { removeDoubleSlashes } from '../routing/useCreatePath';
+import { AuthRedirectResult } from '../types';
+import useAuthProvider from './useAuthProvider';
+
+/**
+ * This hook calls the `authProvider.handleLoginCallback()` method. This is meant to be used in a route called
+ * by an external authentication service (e.g. Auth0) after the user has logged in.
+ * By default, it redirects to the `redirectTo` location, home page if undefined.
+ *
+ * @returns The result of the `handleLoginCallback` call. Destructure as { isLoading, data, error, refetch }.
```

Comment:
`{ isLoading, data, error, refetch }` is not the result of the `handleLoginCallback` call